### ISSUE TITLE
Addon button reappears after upgrade

### ISSUE THREAD

**Issue by timendum** [mozilla/testpilot#2188]
My addon recently updated to 1.1.0 and the test pilot button appeared again in the toolbar. I removed it in the past.

I don't think it's linked to the specific version, but to the upgrade.

I'm on firefox  52.0b5 but I think it happened in the past, after previous addon upgrade.

**Comment by ckprice** [comment]
We haven't shipped anything that could have caused this. This sounds more like a Firefox bug. Maybe @clouserw can test with an older version?

**Comment by SoftVision-PaulOiegas** [comment]
Cosmin had an older version of Test Pilot on one profile, so we fired it up, moved the TxP button in hamburger menu and checked for updates afterwards. The TxP button indeed jumped back in the toolbar after the add-on got updated (see recording - https://goo.gl/WOBhIF).
Isn't this because at TxP update, we uninstall the old add-on and we install the new version afterwards ? If I remember well Les mentioned something like this in a call.

We have tried with an older version of Activity Stream add-on too, since it has the share button for Fx toolbar. Doing the same steps as above, didn't moved back the share button into the toolbar.
